Makefile

```diff
@@ -21,11 +21,13 @@ image: image-magneticod image-magneticow
 # Download dependencies
 ensure:
 	dep ensure -v
+	go get -u github.com/jteeuwen/go-bindata/...
 
 test-magneticod:
 	go test github.com/boramalper/magnetico/cmd/magneticod/...
 
 test-magneticow:
```

Comment:
`test-*` cases are removed in v0.7.0, we use `go test github.com/boramalper/magnetico/...` instead. If you can update your PR for that version, it'd be great. =)

Reply:
Thanks, i've updated my PR!